types/glob/index.d.ts

```diff
@@ -33,9 +33,10 @@ declare namespace G {
         stat?: boolean;
         silent?: boolean;
         strict?: boolean;
-        cache?: { [path: string]: any /* boolean | string | string[] */ };
-        statCache?: { [path: string]: fs.Stats };
-        symlinks?: any;
+        cache?: { [path: string]: boolean | 'DIR' | 'FILE' | string[] };
+        statCache?: { [path: string]: false | { isDirectory(): boolean} | undefined };
```

Comment:
instead of using `fs.Stat` it actually only uses `isDirectory()` and ignores everything else. This typing allows using a subset of `fs.Stat` to pre-fill the cache

Reply:
Do these improvements apply to 5.0 or just 7.1?

Reply:
I honestly don't know. I only checked the implementation of the current major release.
I wouldn't want to change it for the existing typings version as it might someone's build.